### DISCUSSION TITLE
Make multiple organization

### DISCUSSION THREAD

**Discussion by zigzagdev** [community/community#147275] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Question

- make multiple organization but user is only myself case is on free ?

**Comment by shinybrightstar** [comment]
Hi @zigzagdev , we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.

**Reply by zigzagdev** [reply, marked as answer]
@shinybrightstar 

Thanks to response my question and sorry for late to reply.

What i want to say is can i make a multiple organization with my free account without any payment ??
